### PR TITLE
Add Ultraleap submodule

### PR DESCRIPTION
**Pull request by FejZa**
# XRTK - Mixed Reality Toolkit Pull Request

## Overview

Adds the new Ultraleap submodule to core.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -10,3 +10,6 @@
 [submodule "Submodules/Oculus"]
```
Comment (StephenHodgson) [inline .gitmodules:10]: Why is oculus already in here?

**Comment by StephenHodgson** [comment]
When checking out this branch, the ultra leap repo won't clone for me

**Comment by FejZa** [comment]
Same for me, I'll double check

**Comment by StephenHodgson** [comment]
I kinda want to add this nreal and the examples all in one go. Can we close this and do one big PR?

**Comment by FejZa** [comment]
It's working now

**Comment by FejZa** [comment]
Closed

**Pull request closed without merging**